client/src/App.js

```diff
@@ -38,6 +38,7 @@ const App = () => {
           <Grid item xs={12} sm={4}>
             <Form currentId={currentId} setCurrentId={setCurrentId} />
           </Grid>
+          <Map></Map>
```

Comment:
Should we use <Map /> syntax instead of <Map></Map> just a code consistency we can set now. Just for when it has no props or children elements

Reply:
This syntax instead
```suggestion
          <Map />
```

Reply:
I agree